README.md

```diff
@@ -0,0 +1,22 @@
+# matrufsc-crawler
+
+Crawler to extract information from UFSC's CAGR service.
+
+# Intro
+This project aims to generate a JSON with all relevant course data from UFSC's ['Cadastro de Turmas'](https://cagr.sistemas.ufsc.br/modules/comunidade/cadastroTurmas/) page.
+
+# Setup
+This project uses `poetry` as the dependency manager. Install dependencies and run with:
+```
+poetry install
+poetry run matrufsc-crawler ./db.json
+```
```

Comment:
Talvez adicionar
"If you have `pip>=19` installed, it already supports PEP517 build backends, so you can just `pip install` it."